File: 프로그래머스/2/84512. 모음 사전/모음 사전.py
def solution(word):
    vowel = ['A', 'E', 'I', 'O', 'U']
    answer = 0

    for i in range(len(word)):
        idx = vowel.index(word[i])
        for j in range(5-i):
            answer += idx * (5 ** j)
            
    return answer + len(word)

File: 프로그래머스/2/84512. 모음 사전/모음 사전.js
function solution(word) {
    let vowel = ['A', 'E', 'I', 'O', 'U'];
    var answer = 0;
    
    for (let i = 0; i < word.length; i++) {
        const idx = vowel.indexOf(word[i])
        for (let j = 0; j < 5-i; j++) {
            answer += idx * Math.pow(5, j)
        }
    }
    
    return answer + word.length
    
}